glean-core/ffi/glean.h

```diff
@@ -16,6 +16,23 @@
 #include <stdint.h>
 #include <stdlib.h>
 
+/**
+ * A HTTP response code.
+ *
+ * The actual response code is encoded in the lower bits.
+ */
+#define UPLOAD_RESULT_HTTP_STATUS 32768
```

Comment:
I guess cbindgen has something against hex notation?

Reply:
I'm sure internally 0x1 and 1 are just stored as integers, so they loose any formatting information.
It's auto-generated so I don't care.

Reply:
I wasn't worried that it would do the wrong thing at all, I was just curious about how cbindgen worked.